DeepMRSeg/deepmrseg_apply.py

```diff
@@ -43,36 +39,30 @@ def read_flags(argv):
 
 	epilogTxt = '''Examples:
 	
-  ## Apply bmask model on single image (I/O OPTION 1)
-  deepmrseg_downloadmodel --model bmask
-  {prog} --task bmask --inImg sub1_T1.nii.gz --outImg sub1_bmaskseg.nii.gz
+  ## Apply dlicv model on single image (I/O OPTION 1)
+  deepmrseg_downloadmodel --model dlicv
+  {prog} --model dlicv --inImg sub1_T1.nii.gz --outImg sub1_DLICV.nii.gz
```

Comment:
_{prog}_ should be replaced with actual name so users can just copy paste and don't have to think.

Reply:
I agree

Reply:
I checked this. prog here is a variable and it's replaced with the program name, so it's good.